zuul/connection/github.py

```diff
@@ -185,6 +185,20 @@ def _event_pull_request_review(self, request):
         event.type = 'pr-review'
         return event
 
+    def _event_status(self, request):
+        body = request.json_body
+        action = body.get('action')
+        if action == 'pending':
+            return
```

Comment:
Ah. Our design document was mostly about how we'll set up the layout for gates and such, I did' mean for it to narrowly focus what the underlying Zuul code will support.

That said, I get what you're saying about API calls on a pending status. I can accept having it this way in 2.5, and maybe even v3, but we'll let upstream debate that on v3.